zerver/lib/actions.py

```diff
@@ -2385,6 +2385,17 @@ def fetch_stream_subscriber_user_ids(stream: Stream) -> List[int]:
             [(sub.user_profile, stream) for (sub, stream) in subs_to_activate],
             already_subscribed)
 
+def get_available_notification_sounds() -> Iterable[str]:
+    notification_sounds_path = os.path.join(
+        os.path.dirname(__file__), '../../static/audio/notification_sounds')
+    available_notification_sounds = []
+
+    for file_name in os.listdir(notification_sounds_path):
+        if file_name.endswith('.ogg'):
```

Comment:
You can split the filename into base and extension with `os.path.splitext`.

Reply:
@roberthoenig OK, I amended the commit and now use `os.path.splitext`. Thanks!